src/Checkout/CheckoutScreen.js

```diff
@@ -0,0 +1,301 @@
+import React, { useEffect, useState } from 'react';
+import {
+  View, StyleSheet, Alert, ScrollView,
+} from 'react-native';
+import {
+  Text, Button,
+} from 'react-native-paper';
+import PropTypes from 'prop-types';
+import Config from 'react-native-config';
+import Icon from 'react-native-vector-icons/Ionicons';
+
+import CartProduct from '../Cart/CartProduct';
+
+const styles = StyleSheet.create({
+  container: {
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+  titleText: {
```

Comment:
Modify the spacing a little between "Shipping Address" and De Neve Drive text and the size of the icon to match the figma